### ISSUE TITLE
Closing window on Windows throws an exception

### ISSUE THREAD

**Issue by Asuniia** [swiftyapp/swifty#122]
```
TypeError: this.window[t.message] is not a function
    at window:message (C:\Users\lagac\AppData\Local\Programs\Swifty\resources\app.asar\main\index.js:2:7628475)
    at IpcMainImpl.<anonymous> (C:\Users\lagac\AppData\Local\Programs\Swifty\resources\app.asar\main\index.js:2:7632184)
    at IpcMainImpl.emit (node:events:527:28)
    at EventEmitter.<anonymous> (node:electron/js2c/browser_init:161:11014)
    at EventEmitter.emit (node:events:527:28)
```

---

Swifty 0.6.7
Electron 20.3.3
win32 10.0.22000
Locale: fr

**Comment by Asuniia** [comment]
window on close button

**Comment by alchaplinsky** [comment]
Do you get this exception when you close the app window?

**Comment by zlElo** [comment]
I get the same error :( I want to "close" the programm with the close button and this error comes... I have tested it both in a VM and on my laptop and this error always comes up.

**Comment by zlElo** [comment]
VM = Win 10 (outdated)
Laptop = Win 11 (all updates installed)

**Comment by alchaplinsky** [comment]
Ok, some weird behavior on Windows. I need to install a VM to test it out.

**Comment by zshel** [comment]
I get the same issue on Windows 10 21H2 19044.2251
Seems to be consistently reproducible whether I click the minimize, expand or close button.

**Comment by Phil353556** [comment]
Hi,
got same issue when testing on a w10.
uninstall / install and now it is ok 
strange windows

**Comment by zlElo** [comment]
I reinstall all top but it doesn't help anything... But for me it's ok because I change my OS to Ubuntu (ZorinOs) and on Linux runs all!

**Comment by Phil353556** [comment]
same for me.
I tested the fact that with the file "vault.swftx"  backuped, I may use it on another platform. I trust but however I prefer make a test.
During the test I encountered the issue, I copy paste:
```
Unhandled Error
TypeError: this.window[t.message] is not a function
    at window:message (C:/Users/user/AppData/Local/Programs/Swifty/resources/app.asar/main/index.js:2:7967168)
    at IpcMainImpl.<anonymous> (C:/Users/user/AppData/Local/Programs/Swifty/resources/app.asar/main/index.js:2:7970843)
    at IpcMainImpl.emit (node:events:527:28)
    at EventEmitter.<anonymous> (node:electron/js2c/browser_init:165:11237)
    at EventEmitter.emit (node:events:527:28)

```
but as you my thinkpad runs with ubuntu since years.

FYI: I noticed that uninstalling the program do not make a clean uninstall:
-C:\Users\user\AppData\Local    -> sub directory "swifty-updater"   still there with all the contents
-C:\Users\userAppData\Roaming  -> sub directory "Swifty" still there with all the contents  including the file "vault.swftx" 
-C:\Users\userAppData\Local\Packages\Microsoft.Windows.Search_cw5n1h2txyewy\LocalState\AppIconCache\150 with the file "com_electron_swifty"

**Comment by alchaplinsky** [comment]
This should be fixed in v0.6.11
